config/jobs/image-pushing/k8s-staging-csi-secrets-store.yaml

```diff
@@ -0,0 +1,45 @@
+postsubmits:
+  # This is the github repo we'll build from. This block needs to be repeated
+  # for each repo.
+  kubernetes-sigs/secrets-store-csi-driver:
+    # The name should be changed to match the repo name above
+    - name: secrets-store-csi-driver-push-image
+      cluster: k8s-infra-prow-build-trusted
+      annotations:
+        # This is the name of some testgrid dashboard to report to.
+        # If this is the first one for your sig, you may need to create one
+        testgrid-dashboards: sig-auth-secrets-store-csi-driver
+      decorate: true
+      # we only need to run if necessary (e.g.: the version was bumped)
+      run_if_changed: '^docker\/'
+      # this causes the job to only run on the master branch. Remove it if your
+      # job makes sense on every branch (unless it's setting a `latest` tag it
+      # probably does).
+      branches:
+        - ^master$
+      spec:
+        serviceAccountName: deployer # TODO(fejta)- use pusher
+        containers:
+          - image: gcr.io/k8s-testimages/image-builder:v20200213-0032cdb
+            command:
+              - /run.sh
+            args:
+              # this is the project GCB will run in, which is the same as the GCR
+              # images are pushed to.
+              - --project=gcr.io/k8s-staging-csi-secrets-store
```

Comment:
should this just be `--project=k8s-staging-csi-secrets-store`? (for ref: https://github.com/kubernetes/test-infra/blob/master/config/jobs/image-pushing/k8s-staging-kubernetes.yaml#L19)

Reply:
Indeed, it seems like a couple of things have changed in the meanwhile.